src/Domain/DomainCreators/Interval.cpp

```diff
@@ -4,6 +4,7 @@
 #include "Domain/DomainCreators/Interval.hpp"
 
 #include <array>
+#include <charm.h>
```

Comment:
Is this include needed? It's not added in any other domain creators.

Reply:
Unintentional, will remove :)